0 [main] INFO kafka.utils.VerifiableProperties  - Verifying properties
22 [main] INFO kafka.utils.VerifiableProperties  - Property auto.offset.reset is overridden to largest
23 [main] WARN kafka.utils.VerifiableProperties  - Property bootstrap.servers is not valid
23 [main] WARN kafka.utils.VerifiableProperties  - Property destination.topic is not valid
23 [main] INFO kafka.utils.VerifiableProperties  - Property group.id is overridden to space.exploration.mars.rover.curiosity.1536028904396
23 [main] WARN kafka.utils.VerifiableProperties  - Property key.serializer is not valid
23 [main] WARN kafka.utils.VerifiableProperties  - Property metadata.broker.list is not valid
23 [main] WARN kafka.utils.VerifiableProperties  - Property source.topic is not valid
23 [main] WARN kafka.utils.VerifiableProperties  - Property value.serializer is not valid
23 [main] INFO kafka.utils.VerifiableProperties  - Property zookeeper.connect is overridden to localhost:2181
69 [main] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Connecting to zookeeper instance at localhost:2181
76 [ZkClient-EventThread-17-localhost:2181] INFO org.I0Itec.zkclient.ZkEventThread  - Starting ZkClient event thread.
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:host.name=zion-mainframe
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:java.version=10.0.2
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:java.vendor=Oracle Corporation
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:java.home=/usr/lib/jvm/java-11-openjdk-amd64
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:java.class.path=target/mission.control-2.0-shaded.jar
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:java.library.path=/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:java.io.tmpdir=/tmp
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:java.compiler=<NA>
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:os.name=Linux
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:os.arch=amd64
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:os.version=4.15.0-33-generic
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:user.name=sanket
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:user.home=/home/sanket
82 [main] INFO org.apache.zookeeper.ZooKeeper  - Client environment:user.dir=/home/sanket/Documents/workspace/Mission.Control
83 [main] INFO org.apache.zookeeper.ZooKeeper  - Initiating client connection, connectString=localhost:2181 sessionTimeout=6000 watcher=org.I0Itec.zkclient.ZkClient@791d1f8b
97 [main] INFO org.I0Itec.zkclient.ZkClient  - Waiting for keeper state SyncConnected
101 [main-SendThread(localhost:2181)] INFO org.apache.zookeeper.ClientCnxn  - Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
108 [main-SendThread(localhost:2181)] INFO org.apache.zookeeper.ClientCnxn  - Socket connection established to localhost/127.0.0.1:2181, initiating session
112 [main-SendThread(localhost:2181)] INFO org.apache.zookeeper.ClientCnxn  - Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x165a274dc240001, negotiated timeout = 6000
113 [main-EventThread] INFO org.I0Itec.zkclient.ZkClient  - zookeeper state changed (SyncConnected)
122 [main] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], starting auto committer every 60000 ms
140 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], begin registering consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c in ZK
159 [Thread-0] INFO kafka.utils.ZKCheckedEphemeral  - Creating /consumers/space.exploration.mars.rover.curiosity.1536028904396/ids/space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c (is it secure? false)
170 [Thread-0] INFO kafka.utils.ZKCheckedEphemeral  - Result of znode creation is: OK
171 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], end registering consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c in ZK
173 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], starting watcher executor thread for consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c
183 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], begin rebalancing consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c try #0
243 [Thread-0] INFO kafka.consumer.ConsumerFetcherManager  - [ConsumerFetcherManager-1536028904775] Stopping leader finder thread
243 [Thread-0] INFO kafka.consumer.ConsumerFetcherManager  - [ConsumerFetcherManager-1536028904775] Stopping all fetchers
244 [Thread-0] INFO kafka.consumer.ConsumerFetcherManager  - [ConsumerFetcherManager-1536028904775] All connections stopped
244 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Cleared all relevant queues for this fetcher
244 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Cleared the data chunks in all the consumer message iterators
244 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Committing all offsets after clearing the fetcher queues
245 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Releasing partition ownership
262 [Thread-0] INFO kafka.consumer.RangeAssignor  - Consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c rebalancing the following partitions: ArrayBuffer() for topic curiosity_to_earth_5 with consumers: List(space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0)
263 [Thread-0] WARN kafka.consumer.RangeAssignor  - No broker partitions consumed by consumer thread space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0 for topic curiosity_to_earth_5
269 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c selected partitions : 
269 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.consumer.ConsumerFetcherManager$LeaderFinderThread  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread], Starting 
270 [Thread-0] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], end rebalancing consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c try #0
53608 [ZkClient-EventThread-17-localhost:2181] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Topic info for path /brokers/topics/curiosity_to_earth_5 changed to {"version":1,"partitions":{"0":[0]}}, triggering rebalance
53608 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], begin rebalancing consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c try #0
53642 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ConsumerFetcherManager  - [ConsumerFetcherManager-1536028904775] Stopping leader finder thread
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ConsumerFetcherManager$LeaderFinderThread  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread], Shutting down
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ConsumerFetcherManager$LeaderFinderThread  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread], Shutdown completed
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.consumer.ConsumerFetcherManager$LeaderFinderThread  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread], Stopped 
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ConsumerFetcherManager  - [ConsumerFetcherManager-1536028904775] Stopping all fetchers
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ConsumerFetcherManager  - [ConsumerFetcherManager-1536028904775] All connections stopped
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Cleared all relevant queues for this fetcher
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Cleared the data chunks in all the consumer message iterators
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Committing all offsets after clearing the fetcher queues
53643 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Releasing partition ownership
53658 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.RangeAssignor  - Consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c rebalancing the following partitions: ArrayBuffer(0) for topic curiosity_to_earth_5 with consumers: List(space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0)
53658 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.RangeAssignor  - space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0 attempting to claim partition 0
53692 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0 successfully owned partition 0 for topic curiosity_to_earth_5
53697 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], Consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c selected partitions : curiosity_to_earth_5:0: fetched offset = -1: consumed offset = -1
53697 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c_watcher_executor] INFO kafka.consumer.ZookeeperConsumerConnector  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c], end rebalancing consumer space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c try #0
53697 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.consumer.ConsumerFetcherManager$LeaderFinderThread  - [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread], Starting 
53717 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.utils.VerifiableProperties  - Verifying properties
53718 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.utils.VerifiableProperties  - Property client.id is overridden to space.exploration.mars.rover.curiosity.1536028904396
53718 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.utils.VerifiableProperties  - Property metadata.broker.list is overridden to zion-mainframe:9092
53718 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.utils.VerifiableProperties  - Property request.timeout.ms is overridden to 30000
53744 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.client.ClientUtils$  - Fetching metadata from broker BrokerEndPoint(0,zion-mainframe,9092) with correlation id 0 for 1 topic(s) Set(curiosity_to_earth_5)
53750 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.producer.SyncProducer  - Connected to zion-mainframe:9092 for producing
53763 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.producer.SyncProducer  - Disconnecting from zion-mainframe:9092
53829 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.ConsumerFetcherThread  - [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0], Starting 
53850 [space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-leader-finder-thread] INFO kafka.consumer.ConsumerFetcherManager  - [ConsumerFetcherManager-1536028904775] Added fetcher for partitions ArrayBuffer([[curiosity_to_earth_5,0], initOffset -1 to broker BrokerEndPoint(0,zion-mainframe,9092)] )
11865556 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
14988450 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
17489756 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
36379085 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
37684237 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
38786610 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
39646419 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
41084624 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
41816334 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
42555370 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
43610433 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
44317327 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
45738331 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
46420595 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
47862305 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
48737278 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
50524157 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
51369269 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
52889138 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
53795528 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
55164375 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
56280426 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
57413352 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
59663743 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
60815348 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
62724995 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
63564827 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
64502479 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
65422575 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
66049735 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
68557175 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
69518666 [ConsumerFetcherThread-space.exploration.mars.rover.curiosity.1536028904396_zion-mainframe-1536028904724-9146544c-0-0] INFO kafka.consumer.SimpleConsumer  - Reconnect due to error:
java.io.EOFException
	at org.apache.kafka.common.network.NetworkReceive.readFromReadableChannel(NetworkReceive.java:83)
	at kafka.network.BlockingChannel.readCompletely(BlockingChannel.scala:129)
	at kafka.network.BlockingChannel.receive(BlockingChannel.scala:120)
	at kafka.consumer.SimpleConsumer.liftedTree1$1(SimpleConsumer.scala:86)
	at kafka.consumer.SimpleConsumer.kafka$consumer$SimpleConsumer$$sendRequest(SimpleConsumer.scala:83)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1$$anonfun$apply$mcV$sp$1.apply(SimpleConsumer.scala:132)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply$mcV$sp(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.consumer.SimpleConsumer$$anonfun$fetch$1.apply(SimpleConsumer.scala:131)
	at kafka.metrics.KafkaTimer.time(KafkaTimer.scala:33)
	at kafka.consumer.SimpleConsumer.fetch(SimpleConsumer.scala:130)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:108)
	at kafka.consumer.ConsumerFetcherThread.fetch(ConsumerFetcherThread.scala:29)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:102)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:93)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:63)
